src/components/Header/Header.test.js

```diff
@@ -1,80 +1,106 @@
-import Header from "./Header";
-import Courses from "../Courses/Courses";
+import { render, act, screen, fireEvent } from "@testing-library/react";
 import { unmountComponentAtNode } from "react-dom";
-import { render, act } from "@testing-library/react";
-import { BrowserRouter, Route } from "react-router-dom";
-import { useLocation } from "react-router-dom";
+import { BrowserRouter, MemoryRouter, Route } from "react-router-dom";
+
+import Header from "./Header";
 
-let container = HTMLElement;
-let testComponent = (
-  <BrowserRouter>
-    <Header />
-  </BrowserRouter>
-);
+let container = null;
 
 beforeEach(() => {
+  // setup a DOM element as a render target
   container = document.createElement("div");
   document.body.appendChild(container);
 });
 
 afterEach(() => {
+  // cleanup on exiting
   unmountComponentAtNode(container);
   container.remove();
   container = null;
 });
 
-describe("Testing Logo rendering", () => {
-  test("Renders navbar without crashing", () => {
-    const { getByTestId } = render(testComponent);
-    const elm = getByTestId("header-nav");
-    expect(elm).toBeInTheDocument();
-  });
+describe("Header", () => {
+  test("does render", () => {
+    act(() => {
+      render(
+        <MemoryRouter>
+          <Header />
+        </MemoryRouter>
+      );
+    });
 
-  test("Renders the icon text", () => {
-    const { getByText } = render(testComponent);
-    expect(getByText("Experience Management Service")).toBeInTheDocument();
-    expect(getByText("U.S. Department of Defense")).toBeInTheDocument();
+    screen.getByText("Sign In");
+    screen.getByText("Home");
+    screen.getByText("Experience Management Service");
+    screen.getByText("Department of Defense");
```

Comment:
This will need to be moved out to configuration/env-var at some point -- need to keep in mind that this software is for anyone to use.